### PR TITLE
41 rework file structure for frontend and backend support

### PR DESCRIPTION
**Pull request by meganhmoore**
Sorry this is a hefty rework, but mostly I was rearranging things to better fit how we will be developing from now on. 

@kdumais111 I created `static` and `templates` folders where the html, css and js will mostly go from my brief blog reading, and put some placeholder files to test things out but please remove them when you put your work in. 

@JPMartinezClaeys I could only get the django tests to kind of run using `python -m manage test  route_rangers_api.tests` but ideally we will connect it to the other pytests being run and we can configure that in the `pyproject.toml` but since there isn't really anything to test at the moment I am holding off on that. 

once you get models in maybe create a test or two just to make sure its working.

Tests passing:
Screen Shot 2024-04-24 at 4 33 45 PM

### CODE REVIEW

File: app/route_rangers_api/static/map.css
Comment (meganhmoore) [inline app/route_rangers_api/static/map.css:1]: all of these are place holders for your stuff @kdumais111

File: app/route_rangers_api/tests.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pytest
```
[the remark this answers is not in the document]
Comment (JPMartinezClaeys) [reply app/route_rangers_api/tests.py:1]: I'll check this later tomorrow after I set some models into place

**Review by JPMartinezClaeys — APPROVED** [review]
File restructure is always scary but this looks good to me

**Review by jimenasalinas — APPROVED** [review]
Thank you Megan! This should help with the automatic tests :)

**Pull request merged**